### ISSUE TITLE
Any way to manually clear/reset the changelog history?

### ISSUE THREAD

**Issue by summera** [mhenrixon/sidekiq-unique-jobs#568]
**Is your feature request related to a problem? Please describe.**
While initially setting this up, I had changelog enabled. I then disabled the changelog by setting `max_history` to `nil` since I didn't need it anymore, but the old entries remained. 

**Describe the solution you'd like**
It would be nice to have a command to run in the console or a button in the UI to clear the changelog history.

**Comment by mhenrixon** [comment]
That's a great suggestion! I'll make that happen for the next release.

**Comment by mhenrixon** [comment]
It will be released with the next version

**Comment by mhenrixon** [comment]
https://github.com/mhenrixon/sidekiq-unique-jobs/releases/tag/v7.0.1 also makes the reaper start, I added tests for this to avoid such problems in the future.

You should have a `Changelogs` tab in the sidekiq web extensions now @summera

**Comment by summera** [comment]
@mhenrixon that was fast! Thanks a lot. I see that in https://github.com/mhenrixon/sidekiq-unique-jobs/pull/570, the `death_handlers` part was removed from the README:

```ruby
  config.death_handlers << lambda do |job, _ex|
    digest = job["lock_digest"]
    SidekiqUniqueJobs::Digests.new.delete_by_digest(digest) if digest
  end
```

Is that no longer necessary if you are **not** running the reaper? I did notice during my testing that when I raised an exception in a job, `lock_digest` did not seem to be available in that `lambda` but the lock was still cleaned up.
